### DISCUSSION TITLE
Is there a report or list of user edits on the Wiki.js site? (running version 2.5.298)

### DISCUSSION THREAD

**Discussion by JG4321** [requarks/wiki#6483] [Help / Questions] [answered]
Hi everyone,

Our Wiki.js site has three tech writers that add content and make edits daily. Is there a basic report or list of the total page or edits that each user has made for a given time period?

Thanks in advance - Joe Gannon

(Wiki.js version 2.5.298) ( we are using PostGRESQL ver 15 for our database)

**Comment by NGPixel** [comment, marked as answer]
No. You can see the list of last edited pages from the admin dashboard but that's it.

**Comment by JG4321** [comment]
I understand. Thanks for answering so quickly! -Joe Gannon
